src/Plugin/Block/DisplayInformationBlock.php

```diff
@@ -0,0 +1,158 @@
+   */
+  protected $entityDisplayRepository;
+
+  /**
+   * Construct a DisplayInformationBlock instance.
+   *
+   * @param array $configuration
+   *   A configuration array containing information about the plugin instance.
+   * @param string $plugin_id
+   *   The plugin_id for the plugin instance.
+   * @param string $plugin_definition
+   *   The plugin implementation definition.
+   * @param \Drupal\format_strawberryfield\ViewModeResolverInterface $view_mode_resolver
+   *   The SBF View Mode resolver.
+   * @param \Drupal\Core\Entity\EntityDisplayRepositoryInterface $entity_display_repository
+   *   The entity display repository.
+   */
+  public function __construct(array $configuration, $plugin_id, $plugin_definition, ViewModeResolverInterface $view_mode_resolver, EntityDisplayRepositoryInterface $entity_display_repository) {
+    parent::__construct($configuration, $plugin_id, $plugin_definition);
+    $this->viewModeResolver = $view_mode_resolver;
+    $this->entityDisplayRepository = $entity_display_repository;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function create(ContainerInterface $container, array $configuration, $plugin_id, $plugin_definition) {
+    return new static(
+      $configuration,
+      $plugin_id,
+      $plugin_definition,
+      $container->get('format_strawberryfield.view_mode_resolver'),
+      $container->get('entity_display.repository')
+    );
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function build() {
+    $build = [];
+
+    /** @var \Drupal\node\NodeInterface $node */
+    if ($node = $this->getContextValue('node')) {
+      $view_modes = $this->viewModeResolver->getCandidates($node);
+      $selected_view_mode = array_shift($view_modes);
+      $build['info'] = [
+        '#type' => 'details',
+        '#title' => 'Display information',
+        '#open' => TRUE,
+      ];
+
+      // Retrieve the view modes for the entity type to display the label to
+      // the user.
+      $view_mode_definitions = $this->entityDisplayRepository->getViewModes('node');
+
+      $build['info']['selected_view_mode'] = [
+        '#type' => 'html_tag',
+        '#tag' => 'div',
+        '#value' => $this->t('<strong>Selected view mode</strong>: @view_mode',
+          [
+            '@view_mode' => Link::createFromRoute(
+              $view_mode_definitions[$selected_view_mode['view_mode']]['label'] ?? $selected_view_mode['view_mode'],
+              'format_strawberryfield.display_settings',
+              [
+                'node' => $node->id(),
+                'bundle' => $node->bundle(),
+                'view_mode_name' => $selected_view_mode['view_mode'],
+              ]
+            )->toString(),
+          ]
+        ),
+      ];
+      $build['info']['disclaimer'] = [
+        '#type' => 'html_tag',
+        '#tag' => 'div',
+        '#value' => $this->t('<strong>Warning</strong>: Changes on a View Mode will affect every ADO(Node) that is using it.'),
```

Comment:
Let's change this to "Info"

Reply:
@pcambra just a few things pending here before I can merge. The Warning to Info is important so we do not scare the users!